tests/src/main/java/com/hazelcast/stabilizer/tests/map/helpers/IntegerStringEntryListener.java

```diff
@@ -26,34 +26,34 @@
     public int maxDelayMs;
 
     @SuppressWarnings("unused")
-    public EntryListenerImpl() {
+    public IntegerStringEntryListener() {
```

Comment:
Most of the unspecific code creates warnings in default IntelliJ settings because of unchecked assignments.

Also Object is throwing away compile time type checks. So we could easily swap key and value in assignments and verifies (or put a value in there which has nothing to do with key/value types at all).

But you are right on the re-usability. Will change the `EntryListenerImpl` to use generics, so we can use it with any key/value type.